platform/android/java/editor/build.gradle

```diff
@@ -56,15 +57,6 @@ android {
         }
     }
 
-    // Disable 'release' buildtype.
-    // The editor can't be used with target=release only, as debugging tools are then not
-    // included, and it would crash on errors instead of reporting them.
-    variantFilter { variant ->
-        if (variant.buildType.name == "release") {
-            setIgnore(true)
-        }
-    }
-
     applicationVariants.all { variant ->
         variant.outputs.all { output ->
             def suffix = variant.name == "dev" ? "_dev" : ""
```

Comment:
This needs to be updated to update the generated editor binary with the proper name. You should be able to replace it with:
```
def suffix = "_" + variant.name
```